types/staking.go

```diff
@@ -21,18 +25,67 @@ var (
 	DefaultBondDenom = "stake"
 
 	// DefaultPowerReduction is the default amount of staking tokens required for 1 unit of consensus-engine power
-	DefaultPowerReduction = sdkmath.NewIntFromUint64(1000000)
+	DefaultPowerReduction = math.NewIntFromUint64(1000000)
 
 	// PubKeyEd25519Type is ed25519 for type the consensus params validator pub_key_types params
 	PubKeyEd25519Type = "ed25519"
 )
 
 // TokensToConsensusPower - convert input tokens to potential consensus-engine power
-func TokensToConsensusPower(tokens, powerReduction sdkmath.Int) int64 {
+func TokensToConsensusPower(tokens, powerReduction math.Int) int64 {
 	return (tokens.Quo(powerReduction)).Int64()
 }
 
 // TokensFromConsensusPower - convert input power to tokens
-func TokensFromConsensusPower(power int64, powerReduction sdkmath.Int) sdkmath.Int {
-	return sdkmath.NewInt(power).Mul(powerReduction)
+func TokensFromConsensusPower(power int64, powerReduction math.Int) math.Int {
+	return math.NewInt(power).Mul(powerReduction)
+}
+
+// ______________________________________________________________________
+// Delegation & Validator Interfaces are moved here to avoid direct dependency on the staking module
```

Comment:
```suggestion
// Delegation & Validator Interfaces are moved here to avoid direct dependency on the staking module in expected keeper interfaces 
```